batch7crud-roll6/src/main/java/com/lftechnology/batch7crud/model/User.java

```diff
@@ -0,0 +1,42 @@
+package com.lftechnology.batch7crud.model;
+
+public class User {
+	
+	private int id;
+	private String firstName;
+	private String surName;
+	private String userName;
+	private String Password;
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Rename this field "Password" to match the regular expression '^[a-z][a-zA-Z0-9]*$'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS00116)